api/oneclick.py

```diff
@@ -299,8 +299,19 @@ def release_hold(self, patron, pin, licensepool):
         raise CirculationException("Unknown error %s/%s releasing %s.", patron.authorization_identifier, 
             patron_oneclick_id, item_oneclick_id)
 
+    @property
+    def default_circulation_replacement_policy(self):
+        return ReplacementPolicy(
+            identifiers=False,
+            subjects=True,
+            contributions=True,
+            formats=True,
+            analytics=Analytics(self._db),
+        )
```

Comment:
When there _are_ changes to be made, reusing this object instead of creating a new one for every LicensePool also saves a little bit of time.